src/smc-webapp/projects.cjsx

```diff
@@ -525,7 +525,8 @@ class ProjectsStore extends Store
             return 0
         others = []
         for i in v
-            if not i.deleted and (show_hidden or not i.get('users').get(account_id).get('hide'))
+            # deleted projects have a map node " 'deleted': true' standard projects do not have this property
```

Comment:
This is still sloppy:

 - There is a single double quote just sitting there in the middle of the line.   Maybe it is supposed to match the single quote after true?   
 - Also, this seems to be two sentences “deleted projects have...” then another sentence “standard project do not...”, but there is not punctuation between them.  Use a period or semicolon.